src/backend/index.ts

```diff
@@ -71,38 +72,54 @@ ltijs.whitelist(
   '/edusharing-embed/keys'
 )
 
+// since whitelist is not allowing wildcards we ignore the invalidToken event for selected routes
```

Comment:
Oh no! Sad that this does not work. But nice that you found a solution. We might be able to solve it in a different way (similar to serlo-editor-for-edusharing) I'll have a look.